index.html

```diff
@@ -1140,6 +1140,27 @@ <h4>Timing attacks</h4>
       This attack is mitigated by [[[#data-minimization]]], [[[#rate-limiting-change-notifications]]],
       and [[[#same-origin-restriction]]].
     </p>
+    <h4>Cross-site covert channel</h4>
+    <p>
+      In computer security a covert channel creates a capability to transfer information between processes
+      that are not supposed to be allowed to communicate. In modern multi-process web engines in the generic
+      case each window or tab resides in its own process (documents that have the [=same origin=] or sites that
+      have the [=same site=] typically share the same process). Using this API it may be possible to create a
+      cross-site covert channel C where a site A on one tab first writes to the channel C after having
+      manipulated the state of the CPU. Next a site B (that is not same site with site A) on another tab reads
+      from the channel C by using this API to learn when the state of the CPU has changed. This process is
+      repeated as long as the scripts run on both the sites A and B.
+    </p>
+    <p>
+      This attack is in part mitigated by [[[#rate-limiting-change-notifications]]].
+      Implementers are advised to consider additional mitigations for long-running scripts.
+    </p>
+    <div class="note">
+      The longer the scripts run the more information can be transmitted using the proposed cross-site covert channel.
+      For example, if a user is on a video conferencing site and another long-running site that allows for more
```

Comment:
On the other hand, something like an active video conferencing session will push the pressure on the CPU and make it much harder to manipulate in a predictive manner.

Reply:
The same, addressed in https://github.com/w3c/compute-pressure/pull/219/commits/9e8cd0fbda86993a09bfb6f424e27522e30cc73b